apps/gitness/src/styles.css

```diff
@@ -1,3 +1,5 @@
 @import '@harnessio/ui/styles.css';
 /* @import '@harnessio/views/src/styles.css';
 @import '@harnessio/canary/src/styles.css'; */
+
+@import 'highlight.js/styles/atom-one-dark.css';
```

Comment:
Will this css be also intelligently optimized by tailwind? Just wanted to know. This css might have a lot of styles but only few might actually be used in code. We should verify what all goes from it in the final style css for the app.

Reply:
totally true, might be more than one style. I'm not sure if it'll be optimized by tailwind

Reply:
We almost certainly don't want Tailwind to touch this file at all

Reply:
what do u recommend? @knagurski